media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils gnome2-utils xdg
+
+DESCRIPTION="An open-source jukebox for large collections of mp3/ogg/flac files"
+HOMEPAGE="https://gmusicbrowser.org/"
+
+SRC_URI="https://${PN}.org/download/${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="dbus doc extras gstreamer libnotify mplayer"
+
+RDEPEND="dev-lang/perl
+	dev-perl/Gtk2
+	virtual/perl-MIME-Base64
+	|| ( net-misc/wget dev-perl/AnyEvent-HTTP )
+	dbus? ( dev-perl/Net-DBus )
+	gstreamer? ( dev-perl/Glib-Object-Introspection )
+	mplayer? ( || ( media-video/mplayer media-video/mpv ) )
+	!gstreamer? ( !mplayer? (
+			media-sound/alsa-utils
+			media-sound/flac123
+			|| ( media-sound/mpg123 media-sound/mpg321 )
+			media-sound/vorbis-tools
+		)
+	)
+	extras? ( dev-perl/gnome2-wnck )
+	libnotify? ( dev-perl/Gtk2-Notify )"
+DEPEND="sys-devel/gettext
+	doc? ( dev-perl/Text-Markdown )"
```

Comment:
Look for "gettext" in the source code.  If gettext is used for compilation it should be moved to BDEPEND.

Reply:
Thank you for the review, @vaukai !
I've applied all your suggestion above.

About this case - indeed, there is no direct usage of `gettext` in the code.
Only `Makefile` (and sub calls of it) uses `msgfmt`, `msgmerge`, etc.
So I'll move the dep.

BTW I've checked the remaining dependency and found that it also used only in the build process. So I'll move both to `BDEPEND`.

Reply:
Done